src/diffusers/utils/import_utils.py

```diff
@@ -700,6 +700,19 @@ def is_torch_version(operation: str, version: str):
     return compare_versions(parse(_torch_version), operation, version)
 
 
+def is_torch_xla_version(operation: str, version: str):
```

Comment:
can we make sure we can call `is_torch_xla_version()` when it is not installed? currently, I think you will have to run it together with `is_torch_xla_available()`, because the `_torch_xla_version` is not defined otherwise

we can do like this https://github.com/huggingface/diffusers/blob/243d9a49864ebb4562de6304a5fb9b9ebb496c6e/src/diffusers/utils/import_utils.py#L55

Reply:
Nice catch, updated.